src/DashboardService.Mappers/Db/DbBoardMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Db;
+
+public class DbBoardMapper : IDbBoardMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public DbBoardMapper(IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public DbBoard Map(CreateBoardRequest request)
+  {
+    return request is null
```

Comment:
check for null is redundant, this is already checked in the controller

Reply:
+